r-package/grf/bindings/ForestBindings.cpp

```diff
@@ -0,0 +1,34 @@
+#include <Rcpp.h>
+#include "commons/globals.h"
+#include "Eigen/Sparse"
+#include "forest/ForestTrainer.h"
+
+#include "commons/DefaultData.h"
+#include "commons/SparseData.h"
+#include "forest/ForestOptions.h"
+#include "forest/Forest.h"
+#include "RcppUtilities.h"
+#include "serialization/ForestSerializer.h"
+
+
+// [[Rcpp::export]]
+Rcpp::List cpp_join_forests(const Rcpp::List forest_objects,
```

Comment:
Does this need the `cpp` prefix? We don't use it in other bindings files.

Reply:
I think this should be fine because they're namespaced differently (one is nested under `Forest::`)?